quality_of_life/api/endpoints/ranking.py

```diff
@@ -1,11 +1,11 @@
 from fastapi import APIRouter
 
-from quality_of_life.domain.quality_of_life_ranking import get_city_ranking
+from quality_of_life.domain.quality_of_life_ranking import get_ranking_by
 
 
 router = APIRouter()
 
 
 @router.get("/ranking/")
-async def get_ranking(limit: int = 1):
-    return get_city_ranking(limit)
+async def get_ranking(get_by: str, limit: int = 1):
+    return get_ranking_by(get_by, limit)
```

Comment:
I would encourage You to have three endpoints here:
- ranking/city
- ranking/country
- ranking/continent
So You wouldn't have to pass this parameter in request.
Function get_ranking_by is fine